pkg/controller/runtime/deployment/deployment.go

```diff
@@ -59,10 +59,30 @@ func Provision(d *types.Deployment) error {
 
 	// Check deployment is marked for destroy
 	if d.Spec.State.Destroy {
-		// Mark pod for destroy
+
+		if len(pl) == 0 {
+			log.Debugf("controller:deployment:controller:provision: remove deployment %s", d.SelfLink())
```

Comment:
I think prefix can be stored in a constant or variable like it was done in other places

Reply:
Yeah. Do this a bit later, after core code will be migrated.